arcgis-ios-sdk-samples/Content Display Logic/Views/AboutView.swift

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+struct AboutView: View {
+    @Environment(\.presentationMode) private var presentationMode
+    
+    var copyrightText: Text {
+        if #available(iOS 15.0, *) {
+            return Text("Copyright © 2015-\(Date(), format: .dateTime.year()) Esri. All Rights Reserved.")
+        } else {
+            let currentYear = Calendar.current.component(.year, from: Date())
+            return Text("Copyright © 2015-\(currentYear) Esri. All Rights Reserved.")
+        }
+    }
+    
+    var body: some View {
+        NavigationView {
+            List {
+                Section {
+                    VStack {
+                        Image("AppIcon2", label: Text("App icon"))
+                        Text(Bundle.main.name)
+                            .font(.headline)
+                        copyrightText
+                            .font(.footnote)
+                            .foregroundColor(.secondary)
+                    }
+                    .listRowBackground(Color.clear)
+                    .frame(maxWidth: .infinity)
+                }
+                Section {
+                    VersionRow(title: "Version", version: "100.13.0.1")
```

Comment:
Could we just change the version in the project?

![image](https://user-images.githubusercontent.com/2257493/148117075-6c00f4ef-36a4-4af7-a94d-a67c45813367.png)

Reply:
See my comments here: https://github.com/Esri/arcgis-runtime-samples-ios/commit/dcc8d0610467b8ac6f05134b0e67dc4ba16b5f65#commitcomment-62909518 I reverted this change.